### PR TITLE
[server] Sync items

### PR DESCRIPTION
**Pull request by Mnakagawa**
Related: #2210

### CODE REVIEW

File: server/src/DBTablesMonitoring.cc [hunk 1]
```diff
@@ -2777,6 +2777,155 @@ void DBTablesMonitoring::addItemInfoList(const ItemInfoList &itemInfoList)
 	getDBAgent().runTransaction(trx);
 }
 
+static string makeItemIdListCondition(const ItemIdList &idList)
+{
+	string condition;
+	const ColumnDef &colId = COLUMN_DEF_ITEMS[IDX_ITEMS_ID];
+	SeparatorInjector commaInjector(",");
+	condition = StringUtils::sprintf("%s in (", colId.columnName);
```
Comment (cosmo0920) [inline server/src/DBTablesMonitoring.cc]: SQL keywords should be capitalized letters.

in -> IN

File: server/test/Helpers.cc [hunk 2]
```diff
@@ -689,17 +689,6 @@ string makeItemOutput(const ItemInfo &itemInfo)
 	    static_cast<int>(itemInfo.valueType),
 	    itemInfo.unit.c_str());
 
-	// Item category
```
Comment (cosmo0920) [inline server/test/Helpers.cc]: Why did remove these lines? We should assertion about categoryNames in test cases.

Comment (cosmo0920) [inline server/test/Helpers.cc]: Ah, I got it. `items` tables no longer has `categoryNames`.
We should integrate `item_categories` result in this assertion, but this task is not current PR's task.

**Comment by cosmo0920** [comment]
Added test cases are green in my local environment. Awesome!

Let's wait CI results. (Or, should I merge this PR right now?)

**Comment by Mnakagawa** [comment]
Thank you.
Let's wait CI result.

**Comment by cosmo0920** [comment]
https://travis-ci.org/project-hatohol/hatohol/jobs/125463751#L2226 and https://travis-ci.org/project-hatohol/hatohol/jobs/125463750#L2475 are failed.
We should investigate why these test cases are failed.

**Comment by cosmo0920** [comment]
👍

**Pull request merged**